packages/slice-machine/lib/builders/SliceBuilder/SideBar/components/ImagePreview.tsx

```diff
@@ -85,15 +85,14 @@ const ImagePreview: React.FC<ImagePreviewProps> = ({
             {display ? (
               <Fragment>
                 <Flex sx={{ flexDirection: "column" }}>
-                  {!preventScreenshot ? (
```

Comment:
makes sense to sow a disbaled button

Reply:
The screenshot and preview button both works on the same condition.
We decided to have the preview button disabled so I just updated the screenshot button to be disabled as well